lib/middlewares/responseGenerator.js

```diff
@@ -175,19 +193,31 @@ function manageIncomingResponse(id, value) {
 
     var data = synchronousRequests[id];
 
-    if (!data.request.configurationValues) {
-        data.request.configurationValues = {};
-    }
+    if (data && data.request && value && value.length === 1 && value[0].name && value[0].name === 'lazy_op_result') {
+        if (!data.request.configurationValues) {
```

Comment:
Please, feel free not to include this suggestion... :) It is just inheritance from my Mozilla phase... :p What about `data.request.configurationValues = data.request.configurationValues || {}` :)